UefiTestingPkg/UefiTestingPkg.dsc

```diff
@@ -113,29 +126,31 @@
 ###################################################################################################
 [Components]

```

Comment:
Note: Since I was already moving quite a few things around in the component's sections, I went ahead and sorted them.